CODEOWNERS

```diff
@@ -1,9 +1,10 @@
-# These owners will be the default owners for everything in the repo.
+# These owners will be the default owners for all the code in the repo.
 *       @vaguiarpulido
 *       @mariferrarini
+*       @avantikalal
```

Comment:
@vaguiarpulido please keep my name in the last row.

Reply:
Yes, I think I solved this. I don't know why the hell it added all the previous commits I had done... :/